lib/chitragupta/chitragupta.js

```diff
@@ -0,0 +1,57 @@
+const { v4: uuidv4 } = require('uuid');
+
+const CLS = require('./cls');
```

Comment:
Why caps?

Reply:
I referred to [this](https://www.javascripttutorial.net/es6/javascript-const/#:~:text=By%20convention%2C%20the%20constant%20identifiers,values%20can't%20be%20reassigned.&text=However%2C%20variables%20created%20by%20the%20const%20keyword%20are%20%E2%80%9Cimmutable%E2%80%9D.). Also, ran eslint on this. It gave no errors.

Reply:
https://github.com/airbnb/javascript/#naming--uppercase
since we're following airbnb's style guide